pom.xml

```diff
@@ -131,11 +131,16 @@
             <artifactId>jackson-core</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
-            <version>${hadoop.version}</version>
+            <version>3.2.3</version>
             <exclusions>
+                <exclusion>
```

Comment:
any reason why we are excluding here and pinning the version below, if we want to change the version hadoop is using we should put it under dependency management